src/Blob/Helpers/MetadataHelper.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AzureOss\Storage\Blob\Helpers;
+
+/**
+ * @internal
+ */
+class MetadataHelper
```

Comment:
Also keep this final since else people will extend it which will cause more BCs in the future